### PR TITLE
removes unused zookeeper nodes in upgrade

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

File: server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader11to12.java [hunk 1]
```diff
@@ -69,13 +69,22 @@ public class Upgrader11to12 implements Upgrader {
...
-    var rootBase = ZooUtil.getRoot(context.getInstanceID()) + ZROOT_TABLET;
+    var zooRoot = ZooUtil.getRoot(context.getInstanceID());
+    var rootBase = zooRoot + ZROOT_TABLET;
 
     try {
       var zrw = context.getZooReaderWriter();
+
+      // clean up nodes no longer in use
+      zrw.recursiveDelete(zooRoot + ZTRACERS, ZooUtil.NodeMissingPolicy.SKIP);
+      zrw.recursiveDelete(zooRoot + ZCONF_LEGACY, ZooUtil.NodeMissingPolicy.SKIP);
```
Comment (keith-turner) [inline server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader11to12.java:86]: @ctubbsii @EdColeman  is it ok to remove this zookeeper node in the 3.1 upgrade?

**Comment by keith-turner** [comment]
For #4587 was reviewing what is outstanding for the elasticity upgrade I noticed these two unused zookeeper constants.  Seems they can be removed from ZK if they exist in the 3.1 upgrade.

File: server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader11to12.java (hunk 1 above)
Comment (keith-turner) [reply server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader11to12.java:86]: I looked into this a bit and do not think there is anything to actually remove in zookeeper.  Looking at [ConfigPropertyUpgrader](https://github.com/apache/accumulo/blob/0e7417d0369aca85e149563cc52ad7c9108dfe6d/server/base/src/main/java/org/apache/accumulo/server/conf/util/ConfigPropertyUpgrader.java) in 2.1 ZCONF_LEGACY is only used for table and namespace config AND the upgrade sets a boolean to remove those.

Comment (keith-turner) [reply server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader11to12.java:86]: Going to update this PR to only remove the constant.

**Review by dlmarion — APPROVED** [review]

**Pull request merged**